spark-stream-ingest/submit_job.py

```diff
@@ -0,0 +1,162 @@
+"""
+Copyright (c) 2021. Cognitive Scale Inc. All rights reserved.
+
+"""
+import os
+import subprocess
+import yaml
+import sys
+import json
+
+
+def get_runtime_args(config, driver_spec_loc, token):
+    pyspark_args = config['pyspark']
+    options = pyspark_args['options']
+    # os.environ['SPARK_HOME'] = "/Users/bpandey/projects/cortex6/cortex-fabric-examples/spark-stream-ingest/submit/submit/opt/spark"
+    # os.environ['HADOOP_HOME'] 
+    args = [os.environ['SPARK_HOME'] + "/" + pyspark_args['pyspark_bin']]
+    for key in options.keys():
+        val = options[key]
+        if isinstance(val, str):
+            args.append(key)
+            args.append(val)
```

Comment:
The order needs to be switched here.  Check for dictionary first, and then stringify if it's not a dict.  Otherwise, numeric and boolean values will not be set